ipn/localapi/localapi.go

```diff
@@ -2325,6 +2329,71 @@ func (h *Handler) serveDebugLog(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// serveUpdateCheck returns the ClientVersion from Status, which contains
+// information on whether an update is available, and if so, what version,
+// *if* we support auto-updates on this platform. if we don't, this endpoint
+// always returns a ClientVersion saying we're running the newest version.
+// effectively, it tells us whether serveUpdateInstall will be able to install
```

Comment:
s/effectively/Effectively/

Reply:
god I need to get back into capitalizing things